examples/jenkins/master-slave/README.md

```diff
@@ -28,25 +28,28 @@ Steps
 Before you begin, ensure you have created the [default imagestreams](https://docs.openshift.org/latest/install_config/imagestreams_templates.html#creating-image-streams-for-openshift-images) in the openshift namespace.
 
 1. Create new OpenShift project, where the Jenkins server will run:
-```
-$ oc new-project ci
-```
+  ```
+  $ oc new-project ci
+  ```
 
 2. Give the Jenkins Pod [service account](https://docs.openshift.org/latest/admin_guide/service_accounts.html)
    rights to do API calls to OpenShift.  This allows us to do the Jenkins Slave
    image discovery automatically.
-```
-$ oc policy add-role-to-user edit system:serviceaccount:ci:default -n ci
-```
+  ```
+  $ oc policy add-role-to-user edit system:serviceaccount:ci:default -n ci
+  ```
 
 3. Install the provided OpenShift templates:
-```
-# Slave converter (optional):
-$ oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/jenkins/master-slave/jenkins-slave-template.json
-
-# Jenkins master template:
-$ oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/jenkins/master-slave/jenkins-master-template.json
-```
+  ```
+  # Ruby application template
+  $ oc new-app https://raw.githubusercontent.com/openshift/origin/master/examples/jenkins/application-template.json
```

Comment:
this is installing/instantiating a template which is different from installing a template (as the other 2 commands do), so i think this needs to be broken out as a separate step so it's not under "install the provided templates"